src/app/(proper_react)/(redesign)/Shell/Shell.module.scss

```diff
@@ -70,6 +71,7 @@
 
   .page {
     flex: 1 0 auto;
+    overflow: hidden;
```

Comment:
This scares me - I'm afraid that important content won't be reachable. Do you remember why you need this?

Reply:
See https://github.com/mozilla/blurts-server/pull/5671#discussion_r1981288584.

- [ ]